src/utils/log.ts

```diff
@@ -0,0 +1,37 @@
+import * as core from "@actions/core";
+
+export const logDebug = (message: string): void => {
+  core.debug(message);
+};
+
+export const logInfo = (message: string): void => {
+  core.info(message);
+};
```

Comment:
**nit**:

Does `core.debug`/`core.info` take other arguments or return something?

If not, you can directly:

```suggestion
export const logDebug = core.debug;

export const logInfo = core.info;
```